Sites/GD2/Project 1/index.html

```diff
@@ -0,0 +1,71 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="Styles.css" </head> <body>
```

Comment:
Your `link` element is missing the closing `>`:
```suggestion
    <link rel="stylesheet" href="Styles.css">
</head>
<body>
```